mtp_cashbook/templates/disbursements/paper-forms.html

```diff
@@ -15,50 +15,75 @@ <h1 class="heading-xlarge">{% trans 'Paper forms' %}</h1>
       </header>
 
       <p>
-        {% trans 'Prisoners can send money out to people by bank transfer, cheque, cash or postal order.' %}
+        {% trans 'Prisoners can send money out digitally by bank transfer or cheque and (if your prison offers this) by cash or postal order.' %}
       </p>
-      <p>
-        {% trans 'Download and print the correct form, then give it to the prisoner to fill in.' %}
-      </p>
-
     </div>
   </div>
 
+  <h2 class="heading-medium">
+    {% trans 'Digital disbursement form' %}
+  </h2>
   <div class="mtp-paper-form-download grid-row">
-    {% trans 'Download for bank transfer or cheque (PDF)' as link_title %}
-    {% static 'disbursements/disbursement-by-bank-transfer-or-cheque-V3.pdf' as pdf_link %}
+    {% trans 'Bank transfer or cheque (PDF)' as link_title %}
+    {% static 'disbursements/disbursement-by-bank-transfer-or-cheque-V6.pdf' as pdf_link %}
     <div class="column-one-third">
       <a href="{{ pdf_link }}">
-        <img src="{% static 'disbursements/disbursement-by-bank-transfer-or-cheque-V3.jpg' %}" srcset="{% static 'disbursements/disbursement-by-bank-transfer-or-cheque-V3@2x.jpg' %} 2x" alt="{{ link_title }}" />
+        <img src="{% static 'disbursements/disbursement-by-bank-transfer-or-cheque-V6.jpg' %}" srcset="{% static 'disbursements/disbursement-by-bank-transfer-or-cheque-V6@2x.jpg' %} 2x" alt="{{ link_title }}" />
       </a>
     </div>
     <div class="column-two-thirds">
       <a href="{{ pdf_link }}">{{ link_title }}</a>
-      <p>V3 04/2018</p>
+      <p>V6 07/2018</p>
       {% trans 'If the prisoner uses this form:' %}
       <ul class="list list-bullet">
-        <li>{% trans 'Give them an envelope to protect their privacy' %}</li>
         <li>{% trans 'Process the request online in ‘Send payment’' %}</li>
+        <li>{% trans 'Have a colleague confirm the request in ‘Confirm payments’' %}</li>
       </ul>
     </div>
   </div>
 
+  <h2 class="heading-medium">
+    {% trans 'Non-digital disbursement forms' %}
+  </h2>
   <div class="mtp-paper-form-download grid-row">
```

Comment:
so did alex/jane decide they prefer _not_ to have a sentence to tell people to just download one of these?

also, we use to have "Process the request in the usual way" as one of the points, do we not want to point out that these forms should be dealt with in the old way?